src/browser-polyfill.js

```diff
@@ -491,12 +491,27 @@ if (typeof browser === "undefined" || Object.getPrototypeOf(browser) !== Object.
       get: {minArgs: 1, maxArgs: 1},
       set: {minArgs: 1, maxArgs: 1},
     };
+
     // Wrap all "network", "services", "websites" APIs in chrome.privacy.*
-    if (extensionAPIs.privacy) {
+    const privacyTypes = ["network", "services", "websites"];
+    const privacy = {
+      get supported() {
+        return extensionAPIs.privacy;
+      },
+    };
+    for (const privacyType of privacyTypes) {
+      Object.defineProperty(privacy, privacyType, {get: () => {
+        if (extensionAPIs.privacy[privacyType]) {
+          return extensionAPIs.privacy[privacyType];
+        }
+        return {};
+      }});
+    }
+    if (privacy.supported) {
```

Comment:
This is not a lazy getter.

As soon as you dereference the `extensionAPIs.privacy` object, the browser has to initialize the extension bindings for `chrome.privacy` object.

A lazy getter is one where you avoid dereferencing the `extensionAPIs.privacy` object until absolutely necessary.

For example, something like this:

```
apiMetadata.privacy = {
  get network() { // <-- This is a getter. You should probably use Object.defineProperty instead.
    let target = extensionAPIs.privacy && extensionAPIs.privacy.network;
    if (!target) {
      let network = {};
      for (let k of Object.keys(target)) {
       network[k] = settingMetadata;
      }
      this.network = network; // <-- This caches the result, hence the "lazy getter" is run only once.
      return network;
    }
  },
};
```

Reply:
Now I feel so stupid, thanks will update.

Reply:
@Rob--W Updated, hope now it looks better.